### DISCUSSION TITLE
Resolution Download Options

### DISCUSSION THREAD

**Discussion by gman8088** [aclap-dev/video-downloadhelper#1120] [Help and Q&A] [answered]
### Your question

If I use Video Downloader on Firefox with a license and companion app installed, does it provide options for download resolutions for YouTube videos?
Can I usually download in 720p vs 1080p or 360p?
Thanks

### Addon & CoApp version and details

version: 9.1.0.42
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:129.0) Gecko/20100101 Firefox/129.0
{
    "used_history_button": true,
    "download_directory": "C:\\Users\\Gregg\\Desktop",
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": false,
    "open_count_store": 4,
    "successfull_dl": 12,
    "record_download_history": true
}

**Comment by thescrapyarduk** [comment]
All you can do is try it

It should be able to list all available resolutions, if it has them. Some videos I download don't have all available resolutions

**Reply by gman8088** [reply]
Thanks

**Comment by paulrouget** [comment, marked as answer]
If it's not listing all the options you want, in the settings increase the "Variant count" to 6 or 10.
